Goldtree/Goldtree/Program.cs

```diff
@@ -12,19 +12,16 @@ public class LogMode
     {
         public string Type { get; set; }
         public ConsoleColor Color { get; set; }
-        public bool Close { get; set; }
 
-        public LogMode(string Type, ConsoleColor Color, bool Close = false)
+        public LogMode(string Type, ConsoleColor Color)
         {
             this.Type = Type;
             this.Color = Color;
-            this.Close = Close;
         }
 
-        public void Log(string Text, bool NewLine = true)
+        public void Log(string Text)
```

Comment:
Recommend calling this `Write(...)` or `WriteLine(...)` to align with .NET conventions (e.g. `Trace.WriteLine`); Also, param names should be lowercase.